about.html

```diff
@@ -7,15 +7,55 @@
         <link rel="stylesheet" href="style.css" />
     </head>
     <body>
-        <nav>
-            <ul>
+        <nav class="nav-container">
+            <h1 class="nav-title"><a href="https://scrimba.com/learn/frontend" target="_blank">Scrimba 2024</a></h1>
+            <ul class="nav-bar">
                 <li><a href="index.html">Home</a></li>
                 <li><a href="about.html">About</a></li>
                 <li><a href="blog.html">Blog</a></li>
                 <li><a href="resources.html">Resources</a></li>
             </ul>
         </nav>
-        <h1>About Us</h1>
+        <main class="about">
+            <h1 class="about-title">About Us</h1>
```

Comment:
I really like this informative text you have given about Scrimba and the bootcamp! Thanks for the addition.